www-client/elinks/elinks-0.12_pre6-r3.ebuild

```diff
@@ -0,0 +1,166 @@
+
+S="${WORKDIR}/${MY_P}"
+PATCHES=(
+	"${FILESDIR}"/${PN}-9999-parallel-make.patch
+	"${FILESDIR}"/${PN}-0.12_pre5-compilation-fix.patch
+	"${FILESDIR}"/${PN}-0.12_pre5-libressl.patch
+	"${FILESDIR}"/${PN}-0.12_pre5-rand-egd.patch
+	"${FILESDIR}"/${PN}-0.11.2-lua-5.1.patch
+	"${FILESDIR}"/${PN}-0.12_pre5-ruby-1.9.patch
+	"${WORKDIR}"/patches/${PN}-0.12_pre5-js185-1-heartbeat.patch
+	"${WORKDIR}"/patches/${PN}-0.12_pre5-js185-2-up.patch
+	"${WORKDIR}"/patches/${PN}-0.12_pre5-js185-3-histback.patch
+	"${FILESDIR}"/${PN}-0.12_pre5-sm185-jsval-fixes.patch
+	)
+
+src_prepare() {
+	default
+
+	cd "${WORKDIR}" || die
+	eapply "${FILESDIR}"/${PN}-0.10.4.conf-syscharset.diff
+	mv ${PN}-0.10.4.conf ${PN}.conf || die
+	if ! use ftp ; then
+		sed -i -e 's/\(.*protocol.ftp.*\)/# \1/' ${PN}.conf || die
+	fi
+	sed -i -e 's/\(.*set protocol.ftp.use_epsv.*\)/# \1/' ${PN}.conf || die
+	cd "${S}" || die
+
+	# fix lib order in configure check
+	# (these seds are necessary so that @preserved-libs copies are not used)
+	sed -i -e 's:for spidermonkeylib in js .*$:for spidermonkeylib in mozjs185 mozjs js smjs; do:' \
+		configure.in || die
+	# Regenerate acinclude.m4 - based on autogen.sh.
+	cat > acinclude.m4 <<- _EOF || die
+		dnl Automatically generated from config/m4/ files.
+		dnl Do not modify!
+	_EOF
+	cat config/m4/*.m4 >> acinclude.m4 || die
+	sed -i -e 's/-Werror//' configure* || die
+
+	eautoreconf
+}
+
+src_configure() {
+	local myconf=""
+
+	if use debug ; then
+		myconf="--enable-debug"
+	else
+		myconf="--enable-fastmem"
+	fi
+
+	# NOTE about GNUTSL SSL support (from the README -- 25/12/2002)
+	# As GNUTLS is not yet 100% stable and its support in ELinks is not so well
+	# tested yet, it's recommended for users to give a strong preference to OpenSSL whenever possible.
+	if use ssl ; then
+		myconf="${myconf} --with-openssl=${EPREFIX}/usr"
+	else
+		myconf="${myconf} --without-openssl --without-gnutls"
+	fi
+
+	econf \
+		--sysconfdir="${EPREFIX}"/etc/elinks \
+		--enable-leds \
+		--enable-88-colors \
+		--enable-256-colors \
+		--enable-true-color \
+		--enable-html-highlight \
+		$(use_with gpm) \
+		$(use_with zlib) \
+		$(use_with bzip2 bzlib) \
+		$(use_with gc) \
+		$(use_with X x) \
+		$(use_with lua) \
+		$(use_with guile) \
+		$(use_with perl) \
+		$(use_with ruby) \
+		$(use_with idn) \
+		$(use_with javascript spidermonkey) \
+		$(use_with tre) \
+		$(use_enable bittorrent) \
+		$(use_enable nls) \
+		$(use_enable ipv6) \
+		$(use_enable ftp) \
+		$(use_enable gopher) \
+		$(use_enable nntp) \
+		$(use_enable finger) \
+		$(use_enable samba smb) \
+		$(use_enable mouse) \
+		$(use_enable xml xbel) \
+		${myconf}
+}
+
+src_compile() {
+	emake V=1
+}
+
+src_install() {
+	emake V=1 DESTDIR="${D}" install
+
+	insinto /etc/elinks
+	doins "${WORKDIR}"/elinks.conf
+	newins contrib/keybind-full.conf keybind-full.sample
+	newins contrib/keybind.conf keybind.conf.sample
+
+	dodoc AUTHORS BUGS ChangeLog INSTALL NEWS README SITES THANKS TODO doc/*.*
```

Comment:
nice work! One more thing: You can put these in the DOCS= variable.

Reply:
The "doc/*.*" cannot directly be put into DOCS, and putting instead "doc/." into docs would install also all  sub-directories of "doc" including "doc/man".